packages/aws-cdk-lib/aws-dynamodb/README.md

```diff
@@ -1,6 +1,704 @@
+    readCapacity: dynamodb.Capacity.fixed(10),
+    writeCapacity: dynamodb.Capacity.autoscaled({ maxCapacity: 15 }),
+  }),
+  replicas: [
+    {
+      region: 'us-east-1',
+    },
+    {
+      region: 'us-east-2',
+      readCapacity: dynamodb.Capacity.autoscaled({ maxCapacity: 20, targetUtilizationPercent: 50 }),
+    },
+  ],
+});
+```
+
+Further reading:
+https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadWriteCapacityMode.html
+
+## Encryption
+
+All user data stored in a `GlobalTable` is fully encrypted at rest. When creating a new `GlobalTable`, you can select the following table encryption options:
+* AWS owned keys - Default encryption type. The keys are owned by DynamoDB (no additional charge).
+* AWS managed keys - The keys are stored in your account and are managed by AWS KMS (AWS KMS charges apply).
+* Customer managed keys - The keys are stored in your account and are created, owned, and managed by you. You have full control over the KMS keys (AWS KMS charges apply).
+
+The following is an example of how to configure `GlobalTable` encryption using an AWS owned key:
+
+```ts
+const globalTable = new dynamodb.GlobalTable(this, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  encryption: dynamodb.TableEncryptionV2.dynamoOwnedKey(),
+})
+```
+
+The following is an example of how to configure `GlobalTable` encryption using an AWS managed key:
+
+```ts
+const globalTable = new dynamodb.GlobalTable(this, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  encryption: dynamodb.TableEncryptionV2.awsManagedKey(),
+})
+```
+
+When configuring `GlobalTable` encryption using customer managed keys, you must specify the KMS key for the table in the main deployment region as the `tableKey`. A map of `replicaKeyArns` must be provided containing each replica region and the associated KMS key ARN:
+
+```ts
+import * as cdk from 'aws-cdk-lib';
+import * as kms from 'aws-cdk-lib/aws-kms';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'Stack', { env: { region: 'us-west-2' } });
+
+const tableKey = new kms.Key(stack, 'Key');
+const replicaKeyArns = {
+  'us-east-1': 'arn:aws:kms:us-east-1:123456789012:key/g24efbna-az9b-42ro-m3bp-cq249l94fca6',
+  'us-east-2': 'arn:aws:kms:us-east-2:123456789012:key/g24efbna-az9b-42ro-m3bp-cq249l94fca6',
+};
+
+const globalTable = new dynamodb.GlobalTable(stack, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  encryption: dynamodb.TableEncryptionV2.customerManagedKey(tableKey, replicaKeyArns),
+  replicas: [
+    { region: 'us-east-1' },
+    { region: 'us-east-2' },
+  ],
+});
+
+```
+
+Note: When encryption is configured with customer managed keys, you must have a key already created in each replica region.
+
+Further reading:
+https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-mgmt
+
+## Global Secondary Indexes
```

Comment:
Can we have an intermediary header that just contains and explains indexing?

```
## Indexing

DDB can automatically update indexes for you. There are 2 types of indexes:

* Global indexes: they do XXX
* Local indexes: they do YYY

### Global secondary indexes

To do this ...

### Local secondary indexes

To do that ...

```

Reply:
Updated.